### ISSUE TITLE
Select button etc are enabled again after visibility of points layer is toggled

### ISSUE THREAD

**Issue by constantinpape** [hci-unihd/covid-if-annotations#26]
I am [deactivating the select, add and remove button](https://github.com/hci-unihd/covid-if-annotations/blob/master/napari_covid_if_annotations/_key_bindings.py#L34-L40) of the points layer, because they are not necessary or could lead to bugs in the case of add / remove.

However, after toggling the visibility of the points layer, these buttons are enabled again.

I tried to put in [this check in `on_layer_change`](https://github.com/hci-unihd/covid-if-annotations/blob/master/napari_covid_if_annotations/launcher/covid_if_annotations.py#L82-L83), but it does not catch this, because it seems like toggling the visibility is not an event that triggers [viewer.layers.events.changed](https://github.com/hci-unihd/covid-if-annotations/blob/master/napari_covid_if_annotations/launcher/covid_if_annotations.py#L88).

**Comment by kevinyamauchi** [comment]
Ah yeah, sorry about that. Setting the `visible` property also sets the `editable` property, which [triggers the points layer controls to update the buttons](https://github.com/napari/napari/blob/f87e18641c529f6c592553b052805c9d75231b90/napari/_qt/layers/qt_labels_layer.py#L349).

We have [this issue](https://github.com/napari/napari/issues/1273) on the napari github about adding public methods to control which modes the user can access, so a fix is on the roadmap. I will link your issue there so we can be sure to cover it when we make the fix.

Thanks for the feedback!

**Comment by sofroniewn** [comment]
Agreed, we should fix this, and thanks for making/ linking to that issue @kevinyamauchi!
